src/test/java/com/microsoft/graph/functional/OneDriveTests.java

```diff
@@ -85,4 +87,18 @@ public void testDownloadWithCustomRequest() throws IOException {
 		   assertFalse("stream should not be empty", stream.read() == -1);
 		}
 	}
+	@Test
+	public void downloadJsonFileFromOneDrive() throws Exception {
+		final InputStream stream = testBase.graphClient.me()
+		.drive()
+		.root()
+		.itemWithPath("test.json")
```

Comment:
Where is this test.json file located?

Reply:
this unit test assumed it'd already be here. I was unclear on how we arrange our integration testing environment so I didn't want to make the "test too big for no reason". But following the discussion we had on Monday, it's clear that we don't have something clearly defined yet, so I'll keep this in mind and write tests that are a little more self sufficient, even if there is redundancy. I just added arragement at the beginning of the test to upload the file we'll need to download.